.github/workflows/scripts/run-metrics.sh

```diff
@@ -34,6 +34,7 @@ if [ "$GITHUB_EVENT_NAME" == "push" ] || [ "$GITHUB_EVENT_NAME" == "pull_request
 	title "Comparing performance between: $BASE_SHA@trunk (base) and $GITHUB_SHA@$HEAD_BRANCH (head) on WordPress v$WP_VERSION"
 
 	title "##[group]Setting up necessary tooling"
+	corepack enable
```

Comment:
`corepack enable pnpm` we don't need all the shims

Reply:
Good one, thanks! I might have told you earlier, but you rock!

Reply:
Done in 4cfe01f849939df36aea22791d78ae23e7b9301d